test/others/skip-file-rwx-check/run.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+source ../env.sh
+
+make clean
+touch testfile
+chmod +w testfile
+tail --follow testfile &
+tailpid=$!
+if ! "$criu" dump --tree=$tailpid --shell-job --verbosity=4 --log-file=dump.log
+then
+    kill $tailpid
+    echo "Failed to dump process as expected"
+    echo FAIL
+    exit 1
+fi
+chmod -w testfile
+if "$criu" restore --restore-detached --shell-job --verbosity=4 --log-file=restore-expected-fail.log
+then
+    kill $tailpid
+    echo "Unexpectedly restored process with reference to a file who's r/w/x perms changed when --skip-file-rwx-check option was not used"
+    echo FAIL
+    exit 1
+fi
+"$criu" restore --skip-file-rwx-check --restore-detached --shell-job --verbosity=4 --log-file=restore.log
```

Comment:
Printing some error message if the above fails would be helpful if ever we see this test do not pass.

Reply:
Fixed; output is

```
Failed to restore process with reference to a file who's r/w/x perms changed when --skip-file-rwx-check option was used
FAIL
```